guides/source/active_record_migrations.md

```diff
@@ -1020,9 +1023,11 @@ class AddInitialProducts < ActiveRecord::Migration
 end
 ```
 
-However, Rails has a 'seeds' feature that should be used for seeding a database
-with initial data. It's a really simple feature: just fill up `db/seeds.rb`
-with some Ruby code, and run `rake db:seed`:
+For adding initial data after a database is created, Rails has a built in 
+'seeds' feature that makes the process quick and easy. This is especially 
+useful when reloading the database frequently in development and test environments. 
+It's easy to get started with this feature: just fill up `db/seeds.rb` with some 
+Ruby code, and run `rake db:seed`:
```

Comment:
I like this paragraph too :ribbon: – here are some small suggestions:

* How about "To add initial data" rather than "For adding initial data"? 
* And "built-in" rather than "built in"?